docs/application/web/api/6.5/device_api/mobile/tizen/ml_single.html

```diff
@@ -0,0 +1,763 @@
+                </p></li>
+<li class="list"><p>
+ with error type AbortError, if any other error occurs.
+                </p></li>
+</ul>
+</li></ul>
+        </div>
+<div class="example">
+<span class="example"><p>Code example:</p></span><pre name="code" class="examplecode prettyprint">var model = tizen.ml.single.openModel(
+    "/opt/usr/home/owner/media/Documents/mobilenet_v1_1.0_224_quant.tflite");
+
+/* 1 ms is used only for testing, to show TimeoutError */
+/* Use more reasonable value in real life applications */
+model.setTimeout(1);
+
+var tensorsInfo = new tizen.ml.TensorsInfo();
+tensorsInfo.addTensorInfo("tensor", "UINT8", [3, 224, 224]);
+var tensorsData = tensorsInfo.getTensorsData();
+try
+{
+  var tensorsDataOut = model.invoke(tensorsData);
+
+  /* Always call dispose() on no longer needed objects */
+  tensorsDataOut.dispose();
+}
+catch (e)
+{
+  console.log("Error: " + e.message);
+}
+
+/* Always call dispose() on no longer needed objects */
+tensorsData.dispose();
+tensorsInfo.dispose();
+model.close();
+</pre>
+</div>
+<div class="output">
+<span class="title"><p>Output example:</p></span><pre>Error: Failed to invoke: timeout
+</pre>
+</div>
+</dd>
+<dt class="method" id="SingleShot::close">
+<a class="backward-compatibility-anchor" name="::MLSingleShot::SingleShot::close"></a><code><b><span class="methodName">close</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Closes the model and releases memory
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">void close();</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="exceptionlist">
+<p><span class="except">Exceptions:</span></p>
+          <ul class="exception"><li>WebAPIException<ul><li class="list"><p>
+ with error type AbortError, if any error occurs. Calling methods on closed model will trigger <var>AbortError</var>.
+                </p></li></ul>
+</li></ul>
+        </div>
+<div class="example">
+<span class="example"><p>Code example:</p></span><pre name="code" class="examplecode prettyprint">var model = tizen.ml.single.openModel(
+    "documents/mobilenet_v1_1.0_224_quant.tflite", null, null, "TENSORFLOW_LITE", "ANY");
+model.close();
+try
+{
+  console.log(model.input);
+}
+catch (e)
+{
+  console.log("Error: " + e.message);
+}
+</pre>
+</div>
+<div class="output">
+<span class="title"><p>Output example:</p></span><pre>Error: SingleShot object was closed and using it is no longer possible.
+</pre>
+</div>
+</dd>
+</dl>
+</div>
+</div>
+<div class="interface" id="OpenModelSuccessCallback">
+<a class="backward-compatibility-anchor" name="::MLSingleShot::OpenModelSuccessCallback"></a><h3>1.3. OpenModelSuccessCallback</h3>
+<div class="brief">
+ The OpenModelSuccessCallback interface provides a SuccessCallback for the <a href="#MachineLearningSingle::openModel">MachineLearningSingle::openModel()</a> method.
+          </div>
```

Comment:
```suggestion
 The `OpenModelSuccessCallback` interface provides a `SuccessCallback` for the <a href="#MachineLearningSingle::openModel">MachineLearningSingle::openModel()</a> method.
          </div>

Reply:
I have put it into `<em>` tags, as we don't use `` ` `` in our docs.